File: integers/timeDifference/timeDifference.java
import java.util.Scanner;
public class hello {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int tim;
		int ch1=in.nextInt();
		int mi1=in.nextInt();
		int sec1=in.nextInt();
		int ch2=in.nextInt();
		int mi2=in.nextInt();
		int sec2=in.nextInt();
		tim=(ch2-ch1)*3600+(mi2-mi1)*60+(sec2-sec1);
		System.out.print(tim);
		
	}
	
}

File: integers/timeDifference/timeDifference.cpp
#include <iostream>
using namespace std;
int main() {
    int tim,ch1,mi1,sec1,ch2,mi2,sec2;
    cin>>ch1>>mi1>>sec1;
    cin>>ch2>>mi2>>sec2;
    tim=(ch2-ch1)*3600+(mi2-mi1)*60+(sec2-sec1);
    cout<<tim<<endl;
  return 0;
}